torchdrivesim/behavior/iai.py

```diff
@@ -30,19 +32,84 @@ def iai_initialize(location, agent_count, center=(0, 0), traffic_light_state_his
     return agent_attributes, agent_states, response.recurrent_states
 
 
-def iai_drive(location: str, agent_states: Tensor, agent_attributes: Tensor, recurrent_states: List,
-              traffic_lights_states: Optional[dict] = None):
+def iai_conditional_initialize(location, agent_count, agent_attributes=None, agent_states=None, recurrent_states=None, center=(0, 0), traffic_light_state_history=None):
```

Comment:
It might also be used by code outside of `torchdriveenv` if needed, so I keep it here.

Reply:
That is specifically what I would like to avoid. I don't want this version to proliferate, so let's move it to `torchdriveenv`. If it stays here, it will be harder to update later.